### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by Bria222**
started Hello-world project
added h1 heading to index page
connected A.css file to A.html file
added a background-color style to the h1 element

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (ShadyShawkat) [inline .gitignore:2]: - [ ] Kindly add `node_modules` to your `.gitignore` file

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Bria222, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️

- [x] Please add the [HTML-CSS linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse.  Also, please make sure that they are set up correctly and all pass with green checkmarks in the pull request.

- [x] Kindly edit the pull request's description with a summary saying what features did you implement in the pull request. you can edit it using the 3-dots icon on it's top right side and then click edit. (See image below)
image

- [x] Kindly edit the pull request's title to be more descriptive of the feature you implemented in this pull request. as an example you can use "Set up project" or  "Add homepage". You can edit the pull request's title by clicking the "Edit" button on the top right of the page. (See image below)
image

- [x] _Check the comments under the review for the rest of required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bria222** [comment]
@ShadyShawkat I have made the necessary changes

**Comment by ShadyShawkat** [comment]
Hey @Bria222, thank you for implementing the changes.
Please go ahead ad resubmit for a review.

File: A.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* stylelint-disable indentation */
```
Comment (Deepakdanger) [inline A.css:1]: - [ ] This is bad practice of passing linter, you can correct it via running `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` then `npx stylelint "**/*.{css,scss}"`, you can correct this linter error manually also.

File: A.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World || welcome to microverse</title>
+    <link rel="stylesheet" href="A.css">
+</head>
+<body>
+    
+
+    <h1 class="header">Hello Microverse!</h1>
```
Comment (Deepakdanger) [inline A.html:13]: - [ ] As `header` is a tag, Avoid using it as the class name.

**Review by Deepakdanger — COMMENTED** [review]
Hi @Bria222, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Never Merge the PR before Approved.
- [ ] _Check the comment under the review_.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**